src/frontend/apps/e2e/__tests__/app-desk/member-delete.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { expect, test } from '@playwright/test';
+
+import { addNewMember, createTeam, keyCloakSignIn } from './common';
+
+test.beforeEach(async ({ page, browserName }) => {
+  await page.goto('/');
+  await keyCloakSignIn(page, browserName);
+});
+
+test.describe('Members Delete', () => {
+  test('it deletes himself when it is the last owner', async ({
```

Comment:
`it cannot delete...`

Reply:
https://github.com/numerique-gouv/people/pull/147/commits/f4583b546c820e8cd6c5ca74732c6b7c46dee59a